pychromecast/controllers/spotify.py

```diff
@@ -81,10 +82,25 @@ def callback():
         self.waiting.clear()
         self.launch(callback_function=callback)
 
-        # Need to wait for Spotify to be launched on Chromecast completely
-        self.waiting.wait(timeout)
+        counter = 0
+        while counter < (timeout + 1):
+            if self.is_launched:
+                return
+            self.waiting.wait(1)
+            counter += 1
 
         if not self.is_launched:
             raise LaunchError(
                 "Timeout when waiting for status response from Spotify app"
             )
+
+    # pylint: disable=too-many-locals
+    def quick_play(self, **kwargs):
+        """
+        Launches the spotify controller and returns when it's ready.
+        To actually play media, another application using spotify connect is required.
+        """
+        self.access_token = kwargs["access_token"]
+        self.expires = kwargs["expires"]
```

Comment:
```suggestion
    def quick_play(self, access_token, expires):
        """
        Launches the spotify controller and returns when it's ready.
        To actually play media, another application using spotify connect is required.
        """
        self.access_token = access_token
        self.expires = expires
```

Reply:
All the other quick_play functions use purely kwargs, so I can't really switch to positional here. I can, though use `access_token=None, expires=None` and a check afterwards.